src/ontology/phenio-odk.yaml

```diff
@@ -18,20 +18,67 @@ import_group:
     - <http://purl.obolibrary.org/obo/NCBITaxon_Union_*>
     - <http://www.informatics.jax.org/marker/MGI:*>
     - <http://purl.obolibrary.org/obo/CARO_*>
+    - <http://flybase.org/reports/*>
+    - <http://birdgenenames.org/cgnc/*>
+    - <http://purl.obolibrary.org/obo/ECTO_*>
+    - <http://purl.obolibrary.org/obo/ENVO_*>
+    - <http://purl.obolibrary.org/obo/MFOMD_*>
+    - <http://purl.obolibrary.org/obo/MF_*>
+    - <http://purl.obolibrary.org/obo/OBI_*>
+    - <http://purl.obolibrary.org/obo/TS_*>
+    # Reconsider adding these:
+    - <http://purl.obolibrary.org/obo/SO_*>
+    - <http://purl.obolibrary.org/obo/WBls_*>
+    - <http://purl.obolibrary.org/obo/ZFS_*>
+    - <http://purl.obolibrary.org/obo/BSPO_*>
+    - <http://purl.obolibrary.org/obo/MOD_*>
+    - <http://purl.obolibrary.org/obo/PO_*>
+    - <http://purl.obolibrary.org/obo/OGMS_*>
+    - <http://purl.obolibrary.org/obo/CHR_*>
+    - <http://identifiers.org/hgnc/*>
+    - <http://identifiers.org/ncbigene/*>
+    # From PRO:
+    - <http://www.ncbi.nlm.nih.gov/gene/*>
```

Comment:
@caufieldjh all the above, thats why this list is important, are _excluded_ ID spaces. If you feel something is not rightfully _excluded_, we need to properly import it.

Reply:
ah! I see. In that case I will also suggest excluding:
* <http://purl.obolibrary.org/obo/STATO_*>
* DOID
* Wikidata prefixes (WD_Entity and WD_Prop)

Reply:
otherwise I'm happy with the list barring any objection from @kevinschaper